internal/validation/helpers.go

```diff
@@ -78,7 +90,7 @@ func ValidateServiceName(service string) bool {
 }
 
 func validateGatewayName(gateway string) bool {
-	regExp := regexp.MustCompile(`^[0-9a-z-_]+(\/[0-9a-z-_]+|(\.[0-9a-z-_]+)*)$`)
```

Comment:
What is the reason behind removing escaping of "/" in this regex?

Reply:
Goland changed it automatically since it's not needed. I can bring it back.